build/main-branch-sync/sync.sh

```
@@ -28,8 +28,10 @@ do
   ${GIT} checkout release-${CURRENT_VERSION} || ${GIT} checkout -b release-${CURRENT_VERSION}
   echo "* Fast-forward to 'main@{${COMMIT_TIME}}' ..."
   ${GIT} merge --ff-only ${TARGET_COMMIT}
-  echo "* Push to 'release-${CURRENT_VERSION}'"
-  ${GIT} push origin release-${CURRENT_VERSION} || { exit_code=1; echo "* ERROR: Failed to fast forward ${p}"; }
+  for CURRENT_VERSION in ${CURRENT_VERSIONS}; do
+    echo "* Push to 'release-${CURRENT_VERSION}'"
+    ${GIT} push origin release-${CURRENT_VERSION} || { exit_code=1; echo "* ERROR: Failed to fast forward ${p}"; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)